src/main/java/entities/Customer.java

```diff
@@ -0,0 +1,131 @@
+package entities;
+
+import java.sql.Date;
+import java.util.Objects;
+
+public class Customer {
+    private int id;
```

Comment:
Avoid primitive types (use wrappers instead: int -> Integer, boolean -> Boolean, etc) in classes that are associated with DB tables

Reason:
When you want to save new Customer to the DB this object initially **have no** `id` but the value will be `0` as default. Same situation for any other column. For instance if user did not specified age at all his age will be equal to `0` if you use primitive type

Reply:
Please update all entities to use wrappers instead of primitive types